src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -0,0 +1,378 @@
+        grant_request_3 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_2.json')))
+        grant_request_4 = json.load(
+          open(os.path.join('testcases', 'testdata', 'grant_3.json')))
+      
+        # Load GWPZ Record
+        gwpz_record_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'gwpz_record_0.json')))
+  
+        # Load FSS record
+        fss_record_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'fss_record_0.json')))
+  
+        # Load ESC record
+        esc_record_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'esc_sensor_record_0.json')))
+     
+        # Load PPA record
+        ppa_record_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'ppa_record_0.json'))) 
+        # Registration configuration for iteration 1
+        cbsd_records_iter_1 = {'registrationRequests': [device_1, device_2],
+                              'grantRequests': [grant_request_1, grant_request_2]}
+        cbsd_records_iter_2 = {'registrationRequests': [device_3, device_4],
+                              'grantRequests': [grant_request_3, grant_request_4]}
+        protected_entities_iter_1 = {'PPA': [ppa_record_1], 'ESC': [esc_record_1]}
+        protected_entities_iter_2 = {'GWPZ': [gwpz_record_1] , 'FSS': [fss_record_1]}
+
+
+        # SAS Test Harness FAD records
+        sas_th_cbsd_0 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_0.json')))
+        sas_th_cbsd_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_1.json')))
+        sas_th_cbsd_2 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_2.json')))
+        sas_th_cbsd_3 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_3.json')))
+        sas_th_cbsd_4 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_4.json')))
+        sas_th_cbsd_5 = json.load(
+          open(os.path.join('testcases', 'testdata', 'cbsd_5.json')))
+
+        sas_th_0_cbsd_iter_0 = [sas_th_cbsd_0]
+        sas_th_0_cbsd_iter_1 = [sas_th_cbsd_1]
+        sas_th_0_cbsd_iter_2 = [sas_th_cbsd_2]
+        sas_th_1_cbsd_iter_0 = [sas_th_cbsd_3]
+        sas_th_1_cbsd_iter_1 = [sas_th_cbsd_4]
+        sas_th_1_cbsd_iter_2 = [sas_th_cbsd_5]
+    
+        # Load DPA record
+        dpa_0 = json.load(
+          open(os.path.join('testcases', 'testdata', 'dpa_0.json')))
+        dpa_1 = json.load(
+          open(os.path.join('testcases', 'testdata', 'dpa_1.json')))
+
+        sas_harness_0 = {
+                    'name': 'SAS-TH-1',
+                    'url': 'localhost:9001',
+                    'cert_file': 'certs/server_a.cert',
+                    'key_file': 'certs/server_a.key',
+                    'certificateHash': 'dummy',
+                    'cbsdRecord': [sas_th_0_cbsd_iter_0, sas_th_0_cbsd_iter_1, sas_th_0_cbsd_iter_2],
+                    'fileServePath': 'sas_harness_2/'
```

Comment:
Does this need to be configured? Probably it could be randomly generated in the test code without causing any trouble. That way we reduce the chance for the tester to introduce a copy-and-paste error which has two SAS THs using the same data directory.

Reply:
This   configuration is for SAS TH. Parameters are configured based on the inputs   received from the SAS TH development. Which is is under #PR368, we will   update the SAS TH configurations section after #PR386 merge